src/main/java/nextstep/subway/map/application/MapService.java

```diff
@@ -15,6 +19,7 @@
 @Service
 @Transactional
 public class MapService {
+    private static final Logger jsonLogger = LoggerFactory.getLogger("json");
     private LineService lineService;
```

Comment:
```suggestion
    private static final Logger jsonLogger = LoggerFactory.getLogger("json");
    
    private LineService lineService;
```

클래스 변수와 인스턴스 변수 사이에 공백을 추가하면 가독성이 상승합니다~